rs/sns/testing/tests/sns_testing_ci.rs

```diff
@@ -1,20 +1,53 @@
 use candid::{Decode, Encode, Principal};
+use canister_test::Wasm;
+use ic_base_types::{CanisterId, PrincipalId};
+use ic_management_canister_types::CanisterSettings;
+use ic_nervous_system_integration_tests::pocket_ic_helpers::{
+    install_canister_on_subnet, load_registry_mutations, NnsInstaller, STARTING_CYCLES_PER_CANISTER,
+};
+use ic_nns_constants::{LEDGER_INDEX_CANISTER_ID, ROOT_CANISTER_ID};
 use ic_sns_testing::nns_dapp::bootstrap_nns;
 use ic_sns_testing::sns::{
-    create_sns, install_test_canister, upgrade_sns_controlled_test_canister, TestCanisterInitArgs,
+    create_sns_pocket_ic, install_test_canister, upgrade_sns_controlled_test_canister_pocket_ic,
+    TestCanisterInitArgs,
+};
+use ic_sns_testing::utils::{
+    validate_network, validate_target_canister, SnsTestingCanisterValidationError,
+    SnsTestingNetworkValidationError,
 };
+use icp_ledger::Tokens;
 use pocket_ic::PocketIcBuilder;
+use tempfile::TempDir;
 
 #[tokio::test]
-async fn test_sns_testing_pocket_ic() {
+async fn test_sns_testing_basic_scenario() {
```

Comment:
Could you please add at least two comments per test - where's the "preparing the world" and where's the "run code under test" parts?